composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "require": {
         "php": ">=8.1",
-        "robmorgan/phinx": "^2.0",
+        "robmorgan/phinx": "^0.15",
```

Comment:
Note : I am keeping this as `^0.15` and not `1.x-dev` because when migrations is released it cannot install the correact version from the tag. We may want to bump the version according to phinx releases.